plugins/auth-backend/src/providers/google/provider.ts

```diff
@@ -150,11 +154,17 @@ export class GoogleAuthProvider implements OAuthHandlers {
     }
 
     try {
-      const user = await this.identityClient.findUser({
-        annotations: {
-          'google.com/email': profile.email,
-        },
+      const token = await this.tokenIssuer.issueToken({
+        claims: { sub: 'backstage.io/auth-backend' },
```

Comment:
Very unsure if we wanna go here, because this would be the only backend plugin that can use this, but ofc also the only one that will run into this chicken & egg problem. Let's keep it but perhaps tweak as we have a closer look at identities in the auth backend